source/Camera/Advanced/CaptureHDRPrintNormals/CaptureHDRPrintNormals.cpp

```diff
@@ -21,8 +21,10 @@ namespace
         {
             for(int col = (numOfCols / 2 - radius); col < (numOfCols / 2 + radius); col++)
             {
-                std::cout << "Normals (" << row << "," << col << "): ";
-                std::cout << std::setprecision(1) << std::fixed << normals(row, col) << std::endl;
+                std::cout << std::setprecision(2) << std::fixed << "Normals at pixel (" << row << "," << col << "):   "
+                          << "X:" << std::left << std::setfill(' ') << std::setw(8) << normals(row, col).x
+                          << "Y:" << std::setw(8) << normals(row, col).y << "Z:" << std::setw(8) << normals(row, col).z
+                          << std::endl;
```

Comment:
I wanted to use our own toString() method. If we don't like it then we should ask SE to change it.

Reply:
That is a valid reason. I suggest you drop it then.